examples/node/pdf2svg.js

```diff
@@ -12,6 +12,8 @@ require('./domstubs.js');
 
 // Run `gulp dist` to generate 'pdfjs-dist' npm package files.
 var pdfjsLib = require('../../build/dist');
+// Force generate base64 string
+pdfjsLib.PDFJS.disableCreateObjectURL = true
```

Comment:
We are using forceDataSchema in SVGGraphics. Why we need disableCreateObjectURL ?